src/core/symbology-ng/qgsstylev2.cpp

```diff
@@ -135,6 +136,31 @@ bool QgsStyleV2::saveSymbol( QString name, QgsSymbolV2* symbol, int groupid, QSt
   return true;
 }
 
+void QgsStyleV2::exportSymbol( QString folder, QString name, QString format, QSize size )
```

Comment:
why not move the guts of this (the part which does the actual exporting of the symbol) to QgsSymbolV2 and then call that from here? Might be more flexible that way, as you'd then be able to call it from anywhere you got access to a symbol...